tests/unit/models/test_plasma.py

```diff
@@ -1,82 +0,0 @@
-import numpy as np
-
```

Comment:
Was this the only test on the Plasma class? If so, might it be best to reinstate some test for this class later on (not in this pull request)?